OrbitVulkanLayer/CMakeLists.txt

```diff
@@ -26,6 +27,7 @@ add_executable(OrbitVulkanLayerTests)
 target_compile_options(OrbitVulkanLayerTests PRIVATE ${STRICT_COMPILE_FLAGS})
 
 target_sources(OrbitVulkanLayerTests PRIVATE
+        PhysicalDeviceManagerTest.cpp
```

Comment:
This should also be called DeviceManagerTest.cpp now.

Reply:
Thanks!